### PR TITLE
Update docs and examples

### PR DESCRIPTION
**Pull request by catalyst17**

### CODE REVIEW

**Comment by catalyst17** [comment]
* **#101** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/insight/101?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @catalyst17 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Insight
 
-Insight is a blockchain data processing tool designed to fetch, process, and store on-chain data. It provides a solution for indexing blockchain data, facilitating efficient querying of transactions, logs through a simple API.
+Insight is a blockchain data processing tool designed to fetch, process, and store on-chain data. It provides a solution for indexing blockchain data, facilitating efficient querying of blocks, transactions and logs through a simple API.
```
Comment (AmineAfia) [inline README.md:3]: you can't query blocks from insight it's only TXs and events

**Review by AmineAfia — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -383,26 +383,37 @@ failureRecoverer:
 ```
 
 #### Storage
-This application has 3 kinds of strorage. Main, Staging and Orchestrator.
-Each of them takes similar configuration depending on the driver you want to use.
-
+This application has 3 kinds of storage: `main`, `staging` and `orchestrator`.
+Each of them takes similar configuration, sligtly depending on the driver you want to use.
```
Comment (iuwqyir) [inline README.md:387]:
```suggestion
Each of them takes similar configuration, slightly depending on the driver you want to use.
```

**Review by iuwqyir — APPROVED** [review]
Overall looks good, 1 typo

File: README.md (hunk 1 above)
Comment (catalyst17) [reply README.md:3]: yep added this before testing the API, removed now :)

**Pull request merged**